modules/apps/site-navigation/site-navigation-taglib/src/main/java/com/liferay/site/navigation/taglib/internal/util/NavItemUtil.java

```diff
@@ -334,8 +333,6 @@ private static List<SiteNavigationMenuItem> _getSiteNavigationMenuItems(
 	private static final Snapshot<LayoutLocalService>
 		_layoutLocalServiceSnapshot = new Snapshot<>(
 			NavItemUtil.class, LayoutLocalService.class);
-	private static final Snapshot<Portal> _portalSnapshot = new Snapshot<>(
```

Comment:
@W2jx Please remove this field and its reference first, and then convert the rest to Snapshot.